### PR TITLE
feat(build): add rte header files to support building on aarch64

### PR DESCRIPTION
**Pull request by wangzihao3**
Related to: openebs/openebs#1295 openebs/maya#1493

Signed-off-by: wangzihao <wangzihao18@huawei.com>

### CODE REVIEW

File: include/rte_ring.h [hunk 1]
```diff
@@ -97,9 +97,16 @@ extern "C" {
...
-#include <rte_pause.h>
-#include <rte_atomic_generic.h>
-#include <rte_atomic.h>
+#include <generic/rte_atomic.h>
+
+#ifdef __x86_64__
+#include <arch/x86/rte_pause.h>
+#include <arch/x86/rte_atomic.h>
+#endif
+#ifdef __aarch64__
```
Comment (mynktl) [inline include/rte_ring.h]: This should be in form of switch case.` #ifdef __x86_64__ .... #elif defined(__aarch64__) ..  #endif`

**Review by mynktl — CHANGES_REQUESTED** [review]
Thanks for the PR @Wangzihao18 ! Changes looks good. Can you update `rte_ring.h` with switch-case for arch related include? 
Thanks!

File: include/rte_ring.h (hunk 1 above)
Comment (wangzihao3) [reply include/rte_ring.h]: Sure. I will change it now.

Comment (wangzihao3) [reply include/rte_ring.h]: It seems the format is not same. Can I change like this ? `#if defined(__x86_64__) .... #elif defined(__aarch64__) .. #endif` . I am not sure which one is better.

Comment (mynktl) [reply include/rte_ring.h]: `#if defined(__x86_64__) .... #elif defined(__aarch64__) .. #endif`

**Review by mynktl — APPROVED** [review]

**Pull request merged**